packages/oui-progress/README.md

```diff
@@ -0,0 +1,111 @@
+<oui-progress type="info">
+    <oui-progress-threshold value="60"></oui-progress-threshold>
+    <oui-progress-bar type="error" value="80">80%</oui-progress-bar>
+</oui-progress>
+<oui-progress type="info">
+    <oui-progress-threshold value="20"></oui-progress-threshold>
+    <oui-progress-threshold value="50"></oui-progress-threshold>
+    <oui-progress-threshold value="80"></oui-progress-threshold>
+    <oui-progress-bar type="success" value="30">30%</oui-progress-bar>
+    <oui-progress-bar type="error" value="25">25%</oui-progress-bar>
+</oui-progress>
+```
+
+### Label alignment
+
+```html:preview
+<oui-progress type="info">
+    <oui-progress-bar type="success" value="30" label-align="left">Progression: 30%</oui-progress-bar>
+</oui-progress>
+```
+
+### Indeterminate
+
+```html:preview
+<oui-progress type="info" indeterminate="true"></oui-progress>
+<oui-progress type="warning" indeterminate="true"></oui-progress>
+```
+
+## API
+
+### oui-progress
+
+| Attribute               | Type            | Binding | One-time binding | Values                                 | default | Description                            |
+| ----                    | ----            | ----    | ----             | ----                                   | ----    | ----                                   |
+| type                    | String          | @       | yes              | `info`, `success`, `warning`, `error`  |         | Type of the progress component       |
+| max-value               | Number          | @?      | yes              |                                        | 100     | The max value for the progress bar    |
+| indeterminate           | Boolean         | <?      | yes              |                                        | false   | Enable/Disable indeterminate mode. When in this mode, the progress component shows an indefinite wait and the max-value would not have any effect. In this mode, the progress component should not have any other component in it |
+
+### oui-progress-bar
+
+| Attribute               | Type            | Binding | One-time binding | Values                                 | default       | Description                            |
+| ----                    | ----            | ----    | ----             | ----                                   | ----          | ----                                   |
+| type                    | String          | @?      | yes              | `info`, `success`, `warning`, `error`  | parent's type | Type of the progress bar. If not provided, takes the type of the parent |
+| value                   | Number          | <       | no               |                                        |               | The current value for the progress bar                                  |
+| label-align             | String          | @?      | yes              | `left`, `right`                        | right         | The alignment of the label inside the progress bar                            |
```

Comment:
No need, the value is always right aligned (see UI specs)

Reply:
This is removed :)